src/scripts/helpers/README.md

```diff
@@ -0,0 +1,321 @@
+# Scripts Helpers
```

Comment:
Observation: As stated during demo, this readme would be a great addition to the technical docs area. Not to be done with this ticket, but as an additional ticket to this.